platform-tests/src/test/java/org/eclipse/deeplearning4j/nd4j/linalg/blas/BlasTests.java

```diff
@@ -65,6 +64,25 @@ public void pcaTest(Nd4jBackend backend) {
     }
 
 
+    @ParameterizedTest
+    @MethodSource("org.nd4j.linalg.BaseNd4jTestWithBackends#configs")
+    public void pcaTest(Nd4jBackend backend) {
+        double[][] inputArray = { { 1.0, 2.0, 3.0 }, { 11.0, 12.0, 13.0 }, { 21.0, 22.0, 23.0 }, { 31.0, 32.0, 33.0 } };
+        double[][] assertion = new double[][]{
+                {-0.55332, -0.72606, 0.40825},
+                {      -0.57703 ,-0.01936 ,-0.81650},
+                {-0.60073, 0.68735, 0.40825 }
+        };
+        INDArray assertArr = Nd4j.create(assertion);
+        INDArray inputMatrix = Nd4j.create( inputArray );
```

Comment:
In order to verify that PCA.pca_factor does not modify inputMatrix  it would be good to add to test the following:
`INDArray duplicateInput = inputMatrix.dup();`
before the calls to PCA.pca_factor and PCA.pca and then assert after the calls:
`assertEquals(duplicateInput, inputMatrix);`